constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/helper/widget/Carousel.java

```diff
@@ -143,13 +149,24 @@ public void onTransitionCompleted(MotionLayout motionLayout, int currentId) {
             mIndex--;
             System.out.println("decrement index...");
```

Comment:
while here, can you remove the printfs? :)

Reply:
Done✅